src/formatters/ValueFormatter.tsx

```diff
@@ -2,5 +2,5 @@ import React from 'react';
 import { FormatterProps } from '../common/types';
 
 export function ValueFormatter<R>(props: FormatterProps<R>) {
-  return <>{props.row[props.column.key]}</>;
+  return <>{props.row[props.column.key as keyof R]}</>;
```

Comment:
IMO we should remove the default defaultFormatter, and delete `ValueFormatter`. Columns should just do
```tsx
formatter(props) { return <>{props.row.specificValue}</>; }
```

Reply:
we can consider it